### ISSUE TITLE
Export just shows the key name for identification

### ISSUE THREAD

**Issue by alexanderadam** [open-keychain/open-keychain#2338]
I wanted to export my private keys and I have to provide a passphrase for them.
But this is pretty tricky because the only information included for the key is "Enter password for '$key name'". But I have a few keys with my name.

## Expected Behaviour

It would be much better to provide additionally an email address or at least any other identifier for the key. Otherwise I have to _guess_ which passphrase I have to enter.

## Current Behaviour
Currently it's ambiguous.

## Steps to Reproduce (for bugs)
1. Menu "Backup/Restore"
2. Backup "All keys + your own keys"
3. Key-guessing. :wink: 

## Context
I just want to export my keys.

## Your Environment
* Android Version: 7.1.1
* Device Model: Moto Z
* OpenKeychain Version: 5.0.2
* From Google Play or F-Droid?: F-Droid

**Comment by CueHD** [comment]
I agree that it would be better to have more information in this dialogue

If you don't have that many, keys can be exported one at a time after selecting an individual key.
Otherwise, I think that the dialogue likely prompts you in the order that the keys are normally displayed.
